src/test/java/org/folio/ld/fingerprint/test/TestUtil.java

```diff
@@ -375,9 +374,7 @@ public static Resource instance() {
         entry(WITH_NOTE, List.of("with note"))
       ),
       Set.of(INSTANCE),
-      pred2OutgoingResources)
-      .setInventoryId(UUID.fromString("2165ef4b-001f-46b3-a60e-52bcdeb3d5a1"))
-      .setSrsId(UUID.fromString("43d58061-decf-4d74-9747-0e1c368e861b"));
```

Comment:
minor
what about adding this information as a new metadata object instead of removing it?

Reply:
it will cause unused import java.util.UUID;

Reply:
done